packages/components/src/Form/Inputs/Radio/Radio.test.tsx

```diff
@@ -26,32 +26,70 @@
 
 import 'jest-styled-components'
 import React from 'react'
-import { assertSnapshot, renderWithTheme } from '@looker/components-test-utils'
+import { renderWithTheme } from '@looker/components-test-utils'
 import { fireEvent } from '@testing-library/react'
 import { Radio } from './Radio'
 
-test('Radio default', () => {
-  assertSnapshot(<Radio id="radioID" />)
+afterEach(() => {
+  jest.resetAllMocks()
 })
 
-test('Radio checked set to true', () => {
-  assertSnapshot(<Radio checked={true} id="radioID" />)
+test('Renders Radio', () => {
+  const { getByRole } = renderWithTheme(<Radio defaultChecked />)
+  const RadioInput = getByRole('radio')
+
+  expect(RadioInput as HTMLInputElement).toBeInTheDocument()
```

Comment:
All of the tests here-in can be made a lot more terse :)

```suggestion
  renderWithTheme(<Radio defaultChecked />)
  expect(screen.getByRole('radio')).toBeInTheDocument()
```

Also would be good to wrap all of the tests inside a `describe('Radio', () => { ...` to keep them all grouped together in the output.